src/components/Navbar/SettingsDrawer/ReadingExperienceSection.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { shallowEqual, useDispatch, useSelector } from 'react-redux';
+import Combobox from 'src/components/dls/Forms/Combobox';
+import RadioGroup, { RadioGroupOrientation } from 'src/components/dls/Forms/RadioGroup/RadioGroup';
+import { ReadingPreference } from 'src/components/QuranReader/types';
+import {
+  selectReadingPreference,
+  selectReadingPreferences,
+  setReadingPreference,
+  setShowWordByWordTranslation,
+  setShowWordByWordTransliteration,
+} from 'src/redux/slices/QuranReader/readingPreferences';
+
+import { Section, SectionLabel, SectionRow, SectionTitle } from './Section';
+
+// TODO: internationalize labels
+const preferences = [
+  {
+    id: ReadingPreference.Reading,
+    label: 'Reading',
+    value: ReadingPreference.Reading,
+  },
+  {
+    id: ReadingPreference.Translation,
+    label: 'Translation',
+    value: ReadingPreference.Translation,
+  },
+];
+
+// wordByWordOptions will be used as items in Combobox component
+const wordByWordOptions = [
+  {
+    id: 'none',
+    value: 'none',
+    name: 'none',
+    label: 'None',
+  },
+  {
+    id: 'translation',
+    value: 'translation',
+    name: 'translation',
+    label: 'Translation',
+  },
+  {
+    id: 'transliteration',
+    value: 'transliteration',
+    name: 'transliteration',
+    label: 'Transliteration',
+  },
+  {
+    id: 'both',
+    value: 'both',
+    name: 'both',
+    label: 'Both',
+  },
```

Comment:
the displayed value in the combobox should be capital, not sure which field this belongs to

Reply:
props initialInputValue capital case